Sources_v3/StreamChatUI/ChatChannel/MessageComposerMentionCellView.swift

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+class MessageComposerMentionCellView: View {
+    // MARK: Properties
+
+    public lazy var avatarView: AvatarView = AvatarView().withoutAutoresizingMaskConstraints
+    public lazy var usernameLabel: UILabel = UILabel().withoutAutoresizingMaskConstraints
+    public lazy var usernameTagLabel: UILabel = UILabel().withoutAutoresizingMaskConstraints
+    public lazy var suggestionTypeImageView: UIImageView = UIImageView().withoutAutoresizingMaskConstraints
+    private lazy var textStackView: UIStackView = UIStackView().withoutAutoresizingMaskConstraints
+
+    // MARK: - Appearance
+
+    override func setUp() {
+        super.setUp()
+        // TODO: Implement in `CIS-465 Implement SuggestionsViewController`
+    }
+
+    override func defaultAppearance() {
+        super.defaultAppearance()
+        usernameLabel.font = UIFont.preferredFont(forTextStyle: .footnote).bold()
+
+        usernameTagLabel.font = .preferredFont(forTextStyle: .caption1)
+        usernameTagLabel.textColor = .systemGray
+
+        if #available(iOS 13.0, *) {
+            backgroundColor = .systemBackground
+            usernameLabel.textColor = .label
+        } else {
+            backgroundColor = .white
+            usernameLabel.textColor = .black
+        }
+    }
+
+    override func setUpLayout() {
+        addSubview(avatarView)
+        setupLeftImageViewConstraints()
+
+        addSubview(textStackView)
+        setupStack()
+
+        addSubview(suggestionTypeImageView)
+        setupSuggestionTypeImageViewConstraints()
+    }
+
+    // MARK: Lifecycle
+
+    public func setup(
+        with title: String,
+        subtitle: String,
+        userImage: UIImage?,
+        isUserOnline: Bool
+    ) {
+        usernameTagLabel.text = subtitle
+        usernameLabel.text = title
+
+        avatarView.imageView.image = userImage
+        suggestionTypeImageView.image = MessageComposerSuggestionCommandKind.mention.image
+    }
```

Comment:
This is not the best approach. Ideally, you'd set these model values into a variable and call `updateContentIfNeeded()`